pkg/controller/replication/replication_controller.go

```diff
@@ -97,36 +96,21 @@ func (r *ReconcileReplication) Reconcile(request reconcile.Request) (reconcile.R
 	originalReplication := replication.DeepCopy()
 
 	// Fetch the Instance
-	harbor := &registriesv1alpha1.Instance{}
-	ns := types.NamespacedName{
-		Namespace: replication.Namespace,
-		Name:      replication.Spec.ParentInstance.Name,
-	}
-	err = r.client.Get(ctx, ns, harbor)
-	if errors.IsNotFound(err) {
-		replication.Status = registriesv1alpha1.ReplicationStatus{
-			Name:           string(registriesv1alpha1.ReplicationStatusPhaseCreating),
-			Message:        "corresponding harbor instance does not exist",
-			LastTransition: &now,
+	harbor, err := internal.FetchReadyHarborInstance(ctx, replication.Namespace, replication.Spec.ParentInstance.Name, r.client)
+	if err != nil {
+		if err == internal.ErrInstanceNotFound {
+			replication.Status = registriesv1alpha1.ReplicationStatus{Name: string(registriesv1alpha1.ReplicationStatusPhaseCreating)}
+			// Requeue, the instance might not have been created yet
+			return reconcile.Result{RequeueAfter: time.Second * 30}, nil
```

Comment:
The same consistency format thing as above
```suggestion
			return reconcile.Result{RequeueAfter: 30 * time.Second}, nil
```

Reply:
Thanks for pointing out!